Assets/Scripts/Models/Furniture.cs

```diff
@@ -320,12 +325,13 @@ public List<string> ReplaceableFurniture
     public string LocalizationCode { get; private set; }
 
     /// <summary>
-    /// Gets the description of the furniture if there is no localization support.
+    /// Gets the description of the furniture. This is used by localization.
     /// </summary>
     public string UnlocalizedDescription { get; private set; }
 
     /// <summary>
     /// Gets a value indicating whether this furniture is next to any furniture of the same type.
```

Comment:
The comment is still wrong since it make me believe that LinksToNeighbour = true means that is next to a furniture which is not true. Instead it means that if is next to a furniture of the same then, the the sprite will change so that both furniture look connected.